project/src/js/ui/tournament/playerInfo.js

```diff
@@ -0,0 +1,150 @@
+import backbutton from '../../backbutton';
+import helper from '../helper';
+import m from 'mithril';
+import * as xhr from './tournamentXhr';
+import * as utils from '../../utils';
+import i18n from '../../i18n';
+
+export default {
+  controller: function(tournament) {
+    let isOpen = false;
+    const playerData = m.prop();
+
+    function open(p) {
+      xhr.playerInfo(tournament().id, p.name).then((data) => {
```

Comment:
this is only stylistic matters, but I like to write .`then` and `.catch` in a new like as it's more readable I think.

```js
stuff()
.then(doSthAfter)
.catch(handleError);
```